RELEASE-NOTES.md

```diff
@@ -36,18 +38,18 @@
   * `snow snowpark registry list-tags`
 
 ## Fixes and improvements
-* Too long texts in table cells are now wrapped instead of cropped
-* Split global options into separate section in `help`
-* Avoiding unnecessary replace in function/procedure update
-* Added global options to all commands
-* Updated help messages
-* Fixed problem with Windows shortened paths
-* If only one connection is configured, will be used as default
-* Fixed registry token connection issues
-* Fixes in commands belonging to `snow snowpark compute-pool` and `snow snowpark services` groups
+* Too long texts in table cells are now wrapped instead of cropped.
+* Split global options into separate section in `help`.
+* Avoiding unnecessary replace in function/procedure update.
+* Added global options to all commands.
+* Updated help messages.
+* Fixed problem with Windows shortened paths.
+* If only one connection is configured, will be used as default.
+* Fixed registry token connection issues.
+* Fixes in commands belonging to `snow snowpark compute-pool` and `snow snowpark services` groups.
 * Removed duplicated short option names in a few commands by:
-  * Removing `-p` short option for `--password` option for all commands (backward incompatibility affecting all the commands using a connection) (it was conflicting with various options in a few commands)
-  * Removing `-a` short option for `--replace-always` in `snow snowpark function update` command (it was conflicting with short version of `--check-anaconda-for-pypi-deps`)
-  * Removing `-c` short option for `--compute-pool` in `snow snowpark jobs create` (it was conflicting with short version of global `--connection` option)
-  * Removing `-c` short option for `--container-name` in `snow snowpark jobs logs` (it was conflicting with short version of global `--connection` option)
-* Fixed parsing of specs yaml in `snow snowpark services create` command
+  * Removing `-p` short option for `--password` option for all commands (backward incompatibility affecting all the commands using a connection) (it was conflicting with various options in a few commands).
+  * Removing `-a` short option for `--replace-always` in `snow snowpark function update` command (it was conflicting with short version of `--check-anaconda-for-pypi-deps`).
+  * Removing `-c` short option for `--compute-pool` in `snow snowpark jobs create` (it was conflicting with short version of global `--connection` option).
+  * Removing `-c` short option for `--container-name` in `snow snowpark jobs logs` (it was conflicting with short version of global `--connection` option).
+* Fixed parsing of specs yaml in `snow snowpark services create` command.
```

Comment:
Let's try to separate such changes, in case of cp or searching git log it will create confusion